zanzibar/app.py

```diff
@@ -48,5 +49,16 @@ def delete_acl_entry():
 
     return jsonify({'message': 'ACL entry deleted successfully', 'entry': acl_entry.to_dict()}), 200
 
+
+@app.route('/api-key', methods=['GET'])
+def get_app_key():
+    app_key = generate_api_key()
+    return jsonify({'message': 'Api-Key created.', 'api-key': app_key}), 201
+
+@app.route('/dummy', methods=['GET'])
```

Comment:
Can you please remove this dummy method, use for tests only.

Reply:
Sure! :)